guide.adoc

```diff
@@ -0,0 +1,596 @@
+= Understanding how to deploy a simple RESTful service and front-end on OpenShift using Spring Boot and Apache CXF
+
+This document will guide you through the steps to create a simple "Hello World" RESTful web service using
+http://cxf.apache.org/[Apache CXF] and https://spring.io/projects/spring-boot[Spring Boot]. The functionality is
+similar to the application being developed in
+https://spring.io/guides/gs/rest-service/[Spring Boot's guide on building RESTful services]. However, where the Spring
+Boot guide uses Spring-specific annotations to define the endpoint and its method, we will use standard
+https://jcp.org/en/jsr/detail?id=370[JAX-RS] annotations, which make it easier to reuse endpoints implementations across
+servers, should you want to target a different platform than Spring Boot at some point. Moreover, you will also
+understand the steps needed to deploy your application on an OpenShift cluster.
```

Comment:
Perhaps this point should be emphasized more? 
I mean we don't want people to dismiss the guide as just another hello world sample when it has so much insight to offer

Reply:
Good point.